xmlua/document.lua

```diff
@@ -148,6 +176,197 @@ function methods:get_dtd_entity(name)
   return converter.convert_xml_entity(raw_dtd_entity)
 end
 
+
+do  -- C14N methods
+  -- list can be a string (space separated), an array of strings, or nil
+  local function get_namespace_prefix_array(list)
```

Comment:
Could you rename this to `create_xml_string_array` or `create_xml_strings` like `create_xml_node_set`?